### PR TITLE
LBTM

### PR DESCRIPTION
**Pull request by CaseoJKL**

### CODE REVIEW

**Comment by ekwoka** [comment]
Can you add a description and appropriate title?

I think this addresses #3

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (wiktoriavh) [inline .eslintrc:1]: Why?

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (wiktoriavh) [inline .prettierrc.json:1]: Why?

File: vite.config.js [hunk 3]
```diff
@@ -1,7 +1,17 @@
...
 
 export default defineConfig(({}) => {
   return {
-    base: "/",
+    // base: "/",
+    build: {
+      outDir: "build",
+      rollupOptions: {
+        input: {
+          main: resolve(__dirname, "index.html"),
```
Comment (wiktoriavh) [inline vite.config.js:11]: This doesn't solve the issue.

**Comment by wiktoriavh** [comment]
What does LBTM mean?

I need a better title, then I can adjust it. Use this as a guide: https://github.com/wiktoriavh/community-talks/blob/main/.github/CONTRIBUTING.md

If this is for #3, you need to mention that in your PR. You also made changes not relevant to the issue, You also didn't fix anything.

**Review by wiktoriavh — CHANGES_REQUESTED** [review]

**Pull request closed without merging**